src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicLineCommit.cs

```diff
@@ -89,11 +89,19 @@ End Module
 ");
 
             VisualStudio.Editor.PlaceCaret("(", charsOffset: 1);
-            VisualStudio.Editor.SendKeys("x   as   integer", VirtualKey.Tab);
+            VisualStudio.Editor.SendKeys("x   As   integer", VirtualKey.Tab);
```

Comment:
_wild_

Reply:
The <kbd>Tab</kbd> at the end of typing forces completion of `integer` to `Integer`, but the other one was dependent on some completion state that apparently either wasn't active here or timed out somehow. Either way, that wasn't relevant to point of this test so I just made it type the right thing.